include/deal.II/non_matching/mapping_info.h

```diff
@@ -1284,8 +1284,12 @@ namespace NonMatching
     resize_unit_points(n_unit_points);
     resize_data_fields(n_data_points);
 
-    MappingData mapping_data;
-    cell_index = 0;
+    MappingData  mapping_data;
+    MappingData  mapping_data_last_cell;
```

Comment:
Do you mean
```suggestion
    MappingData  mapping_data_previous_cell;
```
(It is not wrong, but I think 'previous' is not ambiguous.)